### DISCUSSION TITLE
Chunk output in console and not inline

### DISCUSSION THREAD

**Discussion by LazDaria** [REditorSupport/vscode-R#1158] [Q&A]
Hi, 

I started to work with R in VSCode, since I usually use it also for python programming and everything is running quite well.
However, I can't seem to figure out how to show chunk output inline (so in the RMD script) instead of the console. This can be done in RStudio via the settings button (Section 3.2.1.1 in https://bookdown.org/yihui/rmarkdown/notebook.html), but I can't find this option in VSCode. is there a way to plot the output of chunks directly in the RMD file? I would like to have this feature as a way of documentation.

Best,
Daria

**Comment by eitsupi** [comment]
This extension does not have that functionality; it is only work in progress in #394.

**Reply by LazDaria** [reply]
Thank you for your answer! In the demo in #394 the output (e.g. plots) is printed into the notebook just below the respective chunk. However, in my case a separate window pops up for each plot and the rest is written to the console. I would like to have the plots in the notebook. Can I change this somewhere?

**Reply by eitsupi** [reply]
#394 is not merged and therefore cannot do so. It is unlikely that this feature will be released anytime soon.

**Reply by LazDaria** [reply]
Ok thank you for the information!

**Comment by guillerminasenn** [comment]
Hi Daria, I think that ipynb notebooks let you plot results of chunks inline.

**Reply by LazDaria** [reply]
Hi Gsenn438,
yes it does, but I'm looking for that option in RMD. 

Thank you though!
